scripts/cleanup

```diff
@@ -34,6 +34,7 @@ killall netmaster
 etcdctl rm -recursive /contiv > /dev/null 2>&1
 etcdctl rm -recursive /contiv.io > /dev/null 2>&1
 curl -X DELETE http://localhost:8500/v1/kv/contiv?recurse > /dev/null 2>&1
+curl -X DELETE http://localhost:8500/v1/kv/contiv.io?recurse > /dev/null 2>&1
```

Comment:
I didn't realize there were two scripts here... This was already handled in scripts/python/testbedApi.py... I will add the fix here as well